contact.html

```diff
@@ -0,0 +1,622 @@
+                                                <a id="button-229994a626" class="cmp-button" href="https://pinterest.com">
+                                                  <span class="cmp-button__icon cmp-button__icon--icon-pinterest" aria-hidden="true">
+
+                                                  </span>
+                                                </a>
+                                              </div>
+                                              <div class="button btn-flipboard aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                                                <a id="button-229994a626" class="cmp-button" href="https://flipboard.com">
+                                                  <span class="cmp-button__icon cmp-button__icon--icon-flipboard" aria-hidden="true">
+
+                                                  </span>
+                                                </a>
+                                              </div>
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+
+                    <div class="container responsivegrid contact-form aem-GridColumn aem-GridColumn--default--none aem-GridColumn--default--5 aem-GridColumn--offset--default--0 aem-GridColumn--phone--none aem-GridColumn--phone--12 aem-GridColumn--offset--phone--0">
+                        <div id="contact-container" class="cmp-container">
+                          <div class="text formtext aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                            <div id="text-cdbe0aaaa2" class="cmp-text">
+                                <p>This is a safe place to learn, grow, and ask quesitons. We will protect your identity and never use this information for marketing purposes.</p>
+                            </div>
+                          </div>
+                          <div class="container responsivegrid form-container aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                            <div id="contact-container" class="cmp-container">
+                              <div class="aem-Grid aem-Grid--12 aem-Grid--default--12">
+                                <div class="container responsivegrid aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                                  <form id="new_form" name="new_form" enctype="multipart/form-data" class="cmp-form aem-Grid aem-Grid--12 aem-Grid--default--12">
+                                    <div class="text full-name aem-GridColumn aem-GridColumn--default--12">
+                                      <div class="cmp-form-text">
+                                        <input class="text-mail" type="text" id="fname" name="firstname" placeholder="Name">
```

Comment:
@Prystal254 Needs a `.cmp-form-text__text` class and `.text-mail` can't go here. Custom classes must be on the root of the component.